TypeScript.py

```diff
@@ -490,7 +498,14 @@ def change_count(self, view):
 
     # called by Sublime when a view receives focus
     def on_activated(self, view):
-        logger.view_debug(view, "enter on_activated")
+        logger.view_debug(view, "enter on_activated " + str(view.id()))
+        if is_special_view(view):
+            if NavToCommand.navto_panel_started:
+                # the current view is the nav to bar
+                NavToCommand.insert_text_finished = False
```

Comment:
Make this a little clearer so the code in easy to understand.  Maybe "# The current view is the QuickPanel.  Set insert_text_finished to false to suppress handling in on_modified".